src/index.ts

```diff
@@ -1,22 +1,33 @@
-import { getInput, setFailed, warning } from '@actions/core';
+import { debug as log, getInput, setFailed, warning } from '@actions/core';
 import { context } from '@actions/github';
 
 import { getData } from './get-data';
 import makePullRequestComment from './make-comment';
 import sendPullRequestData from './send-data';
 
+const debug = (label: string, message: string): void => {
+  log('');
+  log(`[${label.toUpperCase()}]`);
+  log(message);
+  log('');
+};
```

Comment:
referenced [ticket check action](https://github.com/neofinancial/ticket-check-action/blob/81c5641120ae5884356b9c6e9daebfacd7c16d2b/src/ticket-check-action.ts#L12) to accept an additional arg to make debug from actions/core more readable

Reply:
throwback